ui/admin/app/styles/app.scss

```diff
@@ -237,6 +237,15 @@
   }
 }
 
+// No spinning animation for refresh button when user specifies reduced motion
+@media (prefers-reduced-motion: reduce) {
```

Comment:
Do you think we should remove the media query for `@media (prefers-reduced-motion: no-preference)` since that's the default and there's only 2 options for this media query?

Reply:
Good point!! +1 to that question!

Reply:
I agree, really good point. I tested this, like mentioned, by removing the the "no-preference" media query but it doesn't work as expected. Not sure why but specifying "reduce motion" doesn't stop the animation when the media query is removed for "no-preference".

Reply:
Yup I think it's fine, I realized it's most likely some of the other CSS properties are leftover even with the new media query as they don't override the same ones. 

Thanks for investigating!

Reply:
I think this may need more investigation?  [According to MDN, `no-preference` should work across modern browsers, including mobile browsers](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion).

Reply:
This is late response, but below is a vid of what I was referring to. Perhaps my syntax is off somewhere?

https://user-images.githubusercontent.com/107949262/196768735-8671f7f0-8c8e-4b60-a463-0284f621832b.mov

Reply:
Yeah this is my bad, I didn't look too closely and I was thinking we didn't need two queries to accomplish this but forgot this was already accomplished because we had used the `no-preference` option to add animation so by default there was none. We didn't need another query to handle the no animation state.

The issue when you removed the other one was that it was applying it by default now and I think the new selector wasn't targeting the right animation in the first place so it wasn't overriding it with `none`

Reply:
No worries! 
Oh okay. That makes sense thank you!